composer.json

```diff
@@ -12,13 +12,13 @@
         "phpunit/phpunit": "^7.5",
         "mockery/mockery": "^1.3",
         "phpstan/phpstan": "^0.12.42",
-        "friendsofphp/php-cs-fixer": "^2.16"
+        "friendsofphp/php-cs-fixer": "^2.16.1"
```

Comment:
Is there a particular reason for this bump?

Reply:
Check this PR: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4638

It solves the issue when running php-cs-fixer on PHP 7.4 (--prefer-lowest)